third_party/gif_decoder/src/main/java/com/bumptech/glide/gifdecoder/GifDecoder.java

```diff
@@ -195,7 +195,7 @@ public void advance() {
      */
     public int getDelay(int n) {
         int delay = -1;
-        if ((n >= 0) && (n < header.frameCount)) {
+        if (0 <= n && n < header.frameCount) {
```

Comment:
How about "`n` is *between* 0 and `frameCount`"?
It's literally between the two bounds in the code as well.